src/entities/account.entity.ts

```diff
@@ -18,15 +21,19 @@ export class Account extends BaseEntity {
   user: User
 
   @Column({ unique: true })
-  accountNumber: string
+  @Generated('increment')
```

Comment:
Why Generated with increment? We should develop a account number system?

Reply:
bank id + branch + number